### PR TITLE
MINOR: update processor topology test driver

### PR DESCRIPTION
**Pull request by bbejeck**

### CODE REVIEW

**Comment by bbejeck** [comment]
With the `ProcessorTopologyTestDriver` in its current form, users can't test a topology without resorting to using reflection; this is a proposed fix.  I'm not married to it, so I'm open to suggestions.

ping @mjsax @dguy @guozhangwang

**Review by mjsax — APPROVED** [review]
LGTM.

File: streams/src/test/java/org/apache/kafka/streams/InternalTopologyAccessor.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+ * topologies by using the  {@link org.apache.kafka.test.ProcessorTopologyTestDriver}
+ */
+public class InternalTopologyAccessor {
+
+    public static InternalTopologyBuilder getInternalTopologyBuilder(Topology topology) {
+        return topology.internalTopologyBuilder;
+    }
+
+    @Deprecated
+    public static InternalTopologyBuilder getInternalTopologyBuilder(TopologyBuilder topologyBuilder) {
```
Comment (guozhangwang) [inline streams/src/test/java/org/apache/kafka/streams/InternalTopologyAccessor.java]: Do we want to add this deprecated function (ditto below for line 172)? I think moving forward we could avoid adding more tests on the deprecated APIs.

Comment (bbejeck) [reply streams/src/test/java/org/apache/kafka/streams/InternalTopologyAccessor.java]: My thinking was to allow users to write tests, without forcing them to move to the new API, but they will need to migrate eventually, so I'll take it out.

**Review by dguy — APPROVED** [review]
LGTM

**Comment by dguy** [comment]
Thanks @bbejeck, merged to trunk

**Pull request closed without merging**